test/api/jd_by_jetid_test.go

```diff
@@ -293,3 +295,51 @@ func TestGetJetDropsByJetID_negative(t *testing.T) {
 		})
 	}
 }
+
+func TestGetJetDropsByJetID_emptyJetID(t *testing.T) {
+	t.Log("C5457 Get JetDrops by JetID = '*', receive a list containing empty and not empty GetIDs")
+	ts := integration.NewBlockExplorerTestSetup(t).WithHTTPServer(t)
+	defer ts.Stop(t)
+
+	pulsesCount, recordsCount := 5, 2
+	records := testutils.GenerateRecordsFromOneJetSilence(pulsesCount, recordsCount)
+
+	pulses := make(map[int64]bool, pulsesCount)
+	var maxPn pulse.Number = 0
+	jetID := jet.NewIDFromString("")
+	for _, r := range records {
+		pn := r.Record.ID.Pulse()
+		if maxPn < pn {
+			maxPn = pn
+		}
+		pulses[int64(pn)] = false
+		r.Record.JetID = jetID
+	}
+	recordWithNotEmptyJetID := testutils.GenerateRecordInNextPulse(maxPn)
+	inNextPulse := testutils.GenerateRecordInNextPulse(recordWithNotEmptyJetID.Record.ID.Pulse())
+	records = append(records, recordWithNotEmptyJetID)
+	records = append(records, inNextPulse)
+	require.NoError(t, heavymock.ImportRecords(ts.ConMngr.ImporterClient, records))
+	ts.WaitRecordsCount(t, len(records)-1, 5000)
+	c := GetHTTPClient()
+
+	res, err := c.JetDropsByJetID(t, "*", nil)
+	jetIDsAmount := pulsesCount + 1
```

Comment:
Maybe rename `jetIDsAmount`  to `jetDropsAmount` ? I think its about different jetDrops, not different JetIDs